content/ja/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -0,0 +1,674 @@
+---
+title: Podを構成してConfigMapを使用する
+content_template: templates/task
+weight: 150
+card:
+  name: tasks
+  weight: 50
+---
+
+{{% capture overview %}}
+ConfigMapを使用すると、設定をイメージのコンテンツから切り離して、コンテナ化されたアプリケーションの移植性を維持できます。このページでは、ConfigMapを作成し、ConfigMapに保存されているデータを使用してPodを構成する一連の使用例を示します。
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+
+## ConfigMapを作成する
+`kubectl create configmap`コマンドまたは`kustomization.yaml`のConfigMap generatorを使用してConfigMapを作成できます。`kubectl`が`kustomization.yaml`をサポートをしているのは1.14からである点に注意してください。
```

Comment:
```suggestion
`kubectl create configmap`または`kustomization.yaml`のConfigMap generatorを使用してConfigMapを作成できます。`kubectl`が`kustomization.yaml`をサポートをしているのは1.14からである点に注意してください。
```

(ここだけではなく全体的に同じことが言える気がしますが )コマンドって自明なのでいらない気もするんですがどうでしょう

@oke-py @nasa9084 @MasayaAoyama @makocchi-git どう思います？

Reply:
翻訳の方針として **迷ったら原文に従う** ということになってたと思います。
原文だと
`You can use either kubectl create configmap or a ConfigMap generator in kustomization.yaml to create a ConfigMap. Note that kubectl starts to support kustomization.yaml since 1.14.`
の部分ですよね
原文には `command` と書いてないのでここでは書かないでいいと思います。

Reply:
なくてもよいと思います。

Reply:
原文にないのか